src/order-book/api.ts

```diff
@@ -47,6 +48,7 @@ export const ORDER_BOOK_PROD_CONFIG: ApiBaseUrls = {
 export const ORDER_BOOK_STAGING_CONFIG: ApiBaseUrls = {
   [SupportedChainId.MAINNET]: 'https://barn.api.cow.fi/mainnet',
   [SupportedChainId.GNOSIS_CHAIN]: 'https://barn.api.cow.fi/xdai',
```

Comment:
weren't these names changing?

Reply:
No changes yet afaict, just the proposal for a standardization.